cabal.project

```diff
@@ -52,3 +52,15 @@ packages:
 tests: True
 optimization: False
 -- reorder-goals: True
+
+-- https://github.com/haskell-servant/servant-multipart/pull/65
+allow-newer: servant-multipart-api:transformers
+
+allow-newer: servant-js:base
+
+if(impl(ghc >= 9.6.1))
```

Comment:
I added a comment to explain that this is for the doctests.

One issue with `:set -fprint...` is that I think it would error on older GHC versions, so we'd have to gate the GHCi command somehow, maybe with CPP. I prefer putting it here.